test/integration_test/storkctl_migrationschedule_test.go

```diff
@@ -156,23 +158,26 @@ func ValidateMigrationScheduleFromFile(t *testing.T, specFilePath string,
 	if err := dec.Decode(&migrationSchedule); err != nil {
 		return err
 	}
-	if err := ValidateMigrationSchedule(t, migrationSchedule, migrationScheduleName, migrationScheduleNs); err != nil {
+	if err := ValidateMigrationSchedule(t, migrationSchedule, migrationScheduleName, migrationScheduleNs, validateSchedulePolicyInterval); err != nil {
 		return err
 	}
 	return nil
 }
 
 func ValidateMigrationSchedule(t *testing.T, migrationSchedule *storkv1.MigrationSchedule,
-	migrationScheduleName string, migrationScheduleNs string) error {
+	migrationScheduleName string, migrationScheduleNs string, validateSchedulePolicyInterval string) error {
 	// We want to validate if the created schedule policy resource matches our expectations
 	actualMigrationSchedule, err := storkops.Instance().GetMigrationSchedule(migrationScheduleName, migrationScheduleNs)
 	if err != nil {
 		logrus.Errorf("Unable to get the created migration schedule")
 		return err
 	}
 	logrus.Info("Trying to validate the migration schedule")
-	if migrationSchedule.Annotations != nil || actualMigrationSchedule.Annotations != nil {
+	if migrationSchedule.Annotations != nil {
+		require.NotNil(t, actualMigrationSchedule.Annotations, "Expected actualMigrationSchedule not to be nil")
 		require.Equal(t, migrationSchedule.Annotations, actualMigrationSchedule.Annotations, "MigrationSchedule Annotations mismatch")
+	} else {
+		require.Nil(t, actualMigrationSchedule.Annotations, "Expected actualMigrationSchedule not to be nil")
```

Comment:
nit.. My bad. I provided wrong message in my review comment and it got copied here. It should be "Expected actualMigrationSchedule to be nil"

Reply:
Will address